packages/plugins/custom-fields/admin/src/components/ColorPicker/ColorPickerInput/index.js

```diff
@@ -0,0 +1,198 @@
+import React, { useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Stack } from '@strapi/design-system/Stack';
+import { Typography } from '@strapi/design-system/Typography';
+import { Flex } from '@strapi/design-system/Flex';
+import { Box } from '@strapi/design-system/Box';
+import { BaseButton } from '@strapi/design-system/BaseButton';
+import { FocusTrap } from '@strapi/design-system/FocusTrap';
+import { Field, FieldHint, FieldError, FieldLabel, FieldInput } from '@strapi/design-system/Field';
+import { Popover } from '@strapi/design-system/Popover';
+import CarretDown from '@strapi/icons/CarretDown';
+import { useIntl } from 'react-intl';
+import { HexColorPicker } from 'react-colorful';
+
+import getTrad from '../../../utils/getTrad';
+
+const ColorPreview = styled.div`
+  border-radius: 50%;
+  width: 20px;
+  height: 20px;
+  margin-right: 10px;
+  background-color: ${(props) => props.color};
+  border: 1px solid rgba(0, 0, 0, 0.1);
+`;
+
+const ColorPicker = styled(HexColorPicker)`
+  && {
+    width: 100%;
+    aspect-ratio: 1.5;
+  }
+
+  .react-colorful__pointer {
+    width: ${({ theme }) => theme.spaces[3]};
+    height: ${({ theme }) => theme.spaces[3]};
+  }
+
+  .react-colorful__saturation {
+    border-radius: ${({ theme }) => theme.spaces[1]};
+    border-bottom: none;
+  }
+
+  .react-colorful__hue {
+    border-radius: 10px;
+    height: ${({ theme }) => theme.spaces[3]};
+    margin-top: ${({ theme }) => theme.spaces[2]};
+  }
+`;
+
+const ColorPickerToggle = styled(BaseButton)`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  svg {
+    width: ${({ theme }) => theme.spaces[2]};
+    height: ${({ theme }) => theme.spaces[2]};
+  }
+
+  svg > path {
+    fill: ${({ theme }) => theme.colors.neutral500};
+    justify-self: flex-end;
+  }
+`;
+
+const ColorPickerPopover = styled(Popover)`
+  padding: ${({ theme }) => theme.spaces[1]};
```

Comment:
FYI on firefox the handles are slightly cropped out as they move out of the container
<img width="451" alt="Screenshot 2022-09-01 at 11 05 40" src="https://user-images.githubusercontent.com/8087692/187880721-b344251b-0317-4802-8c75-f73bd5b60cca.png">

Reply:
Yeah I think I have the same thing on chrome. The only thing I can think to do is increase the padding of the popover so it isn't cut off, could we do that @maevalienard? I don't really have much control over the pointer's  position unfortunately
![Screenshot 2022-09-01 at 12 01 45](https://user-images.githubusercontent.com/26598053/187888110-8352f2d2-0097-4f02-8251-f226d64942d5.png)

Reply:
Otherwise we may be able to hack something using CSS I think, like with negative negative margins, z-index or something

Reply:
Yes that would work, you could add 4 additional px so that would fit the grid of the DS :)

Reply:
ok great already updated :)